README.md

```diff
@@ -52,6 +54,10 @@ The project is divided into folders based on language. Even though the project i
   - Most(read:some) modules are covered by unit tests. Tests are placed in the spec folder.
 - Styling is found in the stylesheets folder. For styling we use [less](https://lesscss.org/). Check out their documentation for getting up to speed on how this differs from traditional css.
 
+## Committing changes
+
+You need to be a member of the Liquipedia organization before you are allowed to push to this repository. In most workflows, you will make a fork of this repository to your own repository, and request a merge request from there.
```

Comment:
several contributors can branch instead of fork
(but we can not merge into main)